search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -56,13 +57,25 @@
   [context rfms]
   (map #(rfm->umm-collection context %) rfms))
 
+(defn- refresh-and-get-collections-from-cache
+   "Force the collection cache to be populated and then get all the collections from it."
+   [context]
+   (metadata-cache/refresh-cache context)
+   (let [rfms (metadata-cache/all-cached-revision-format-maps context)]
+     (if (nil? (seq rfms))
```

Comment:
Thanks. I thought it didn't look quite right. I will change it to (if (seq rams) ....   Done.